### DISCUSSION TITLE
Att@ck Navigator on SO 2.3.120 is using version 7 of the framework (current version is 11)

### DISCUSSION THREAD

**Discussion by btdewey** [Security-Onion-Solutions/securityonion#8401] [Unsupported Versions] [answered]
I was going to use the Attack Navigator on 2.3.120 and quickly realized it's an old version.   Are there any plans to update the navigator version any time soon?  I was also noticing that playbook did not display the sub-technique decimal point and was wondering of playbook and navigator were intentionally on a specific version of Att@ck.  Would there be any issues updating the navigator without making any changes to playbook?  I may try and update navigator and if playbook will work fine with a newer version of navigator.  I know there is a script that creates the default navigator layer and I don't think it would be hard to update to the newer layer format.

**Comment by xfaith** [comment, marked as answer]
In .130 release notes it states UPGRADE: attack-navigator v4.6.4.   Current newest release of Attack-Navigator that I believe SO uses https://github.com/mitre-attack/attack-navigator/releases  (4.6.5).    

I see references to v11 on the site, dont use the MITRE that much currently so not sure why the two version numbers.

**Reply by btdewey** [reply]
Appreciate the quick response.  Was not finding where it talked about the mitre navigator in the release notes.
